### PR TITLE
Fix UpdateSelenium.ps1 script 

### LINKED ISSUE #798: Webdriver fails to update
# 🐛 Summary #

The webdriver for chrome fails to update when executing UpdateSelenium.ps1.

## To reproduce ##

1. Exucute UpdateSelenium.ps1
1. Verify driver is updated

## Expected behavior ##

Expected updated driver to be compatable with installed chrome version

### PR DESCRIPTION
**Pull request by crutchfield**
#  #

## 🗣 Description ##

Retry logic was repreventing download of updated driver. 

## 💭 Motivation and context ##

Current driver aligned with install Chrome web browser needed to conduct functional tests.
Closes #798 

## 🧪 Testing ##

Installed fresh Selenium with outdated drive and updated (Driver updated)
Updated a current driver (No change)

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [X] Tests have been added and/or modified to cover the changes in this PR.
- [X] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by tkol2022 — APPROVED** [review]
### Testing results
- Tested against a freshly installed version of Selenium that had a chromedriver.exe dated with 10/2023. After running the new UpdateSelenium script I validated that the driver file was updated to a version dated 12/2023.
- Tested against a version of Selenium that had no existing chromedriver.exe in the expected path to make sure that the script still works.

**Comment by crutchfield** [comment]
> Not an official review, but some observations nonetheless.

Yea, it was a style guide mess.  I might as well clean it up a bit.

File: Testing/Functional/SmokeTest/UpdateSelenium.ps1 [hunk 1]
```diff
@@ -8,21 +8,20 @@ This script installs the required web driver needed for the current Chrome Brows
...
+.\UpdateSelenium.ps1 -WebDriversPath $WebDriverPath
+
 #>
 param (
-    $registryRoot        = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths",
-    $chromeRegistryPath  = "$registryRoot\chrome.exe",
-    $webDriversPath      = "C:\Program Files\WindowsPowerShell\Modules\Selenium\3.0.1\assemblies",
-    $chromeDriverPath    = "$($webDriversPath)\chromedriver.exe"
+    $RegistryRoot = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths",
+    $WebDriversPath = "C:\Program Files\WindowsPowerShell\Modules\Selenium\3.0.1\assemblies"
```
[the remark this answers is not in the document]
Comment (crutchfield) [reply Testing/Functional/SmokeTest/UpdateSelenium.ps1]: Added change to lookup selenium path instead of defaulting input parameter

**Review by nanda-katikaneni — APPROVED** [review]

**Comment by crutchfield** [comment]
@schrolla Do you want to see if comments are resolved?  I can resolve if not.

**Comment by crutchfield** [comment]
@nanda-katikaneni Ready to merge

**Pull request merged**